sharktank/conftest.py

```diff
@@ -256,6 +272,16 @@ def get_model_artifacts(request: FixtureRequest):
     model_path["llama3_405b_fp8_model_path"] = set_fixture_from_cli_option(
         request, "--llama3-405b-fp8-model-path", "llama3_405b_fp8_model"
     )
+    model_path["google__t5_v1_1_small_fp32_model_path"] = set_fixture_from_cli_option(
```

Comment:
Do we need 2 separate model_path flags for small vs xxl models instead of just calling this twice from our tests?